### PR TITLE
Explanation for beginner programmers

### PR DESCRIPTION
**Pull request by StefKors**
I think adding the enctype="multipart/form-data" to the form isn't something a beginner programmer knows about. This might improve the experience of using Multer and reduce issues being raised.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ $ npm install --save multer
 Multer adds a `body` object and a `file` or `files` object to the `request` object. The `body` object contains the values of the text fields of the form, the `file` or `files` object contains the files uploaded via the form.
 
 Basic usage example:
+Don't forget the `enctype="multipart/form-data"` in your form.
+
+```html
+<form class="" action="/cool-profile" method="post" enctype="multipart/form-data">
+	<input type="file" id="image" name="image_upload" />
```
Comment (LinusU) [inline README.md]: could you change to two space indentation?

File: README.md [hunk 2]
```diff
@@ -23,6 +23,13 @@ $ npm install --save multer
 Multer adds a `body` object and a `file` or `files` object to the `request` object. The `body` object contains the values of the text fields of the form, the `file` or `files` object contains the files uploaded via the form.
 
 Basic usage example:
+Don't forget the `enctype="multipart/form-data"` in your form.
+
+```html
+<form class="" action="/cool-profile" method="post" enctype="multipart/form-data">
```
Comment (LinusU) [inline README.md]: maybe remove the empty class?

File: README.md (hunk 1 above)
Comment (LinusU) [reply README.md]: Maybe remove the `id`?

Comment (LinusU) [reply README.md]: We should probably match the name to the code snippet below

**Comment by StefKors** [comment]
I've added the suggested formatting updates, let me know if there are any other things to change.

File: README.md [hunk 3]
```diff
@@ -23,6 +23,13 @@ $ npm install --save multer
 Multer adds a `body` object and a `file` or `files` object to the `request` object. The `body` object contains the values of the text fields of the form, the `file` or `files` object contains the files uploaded via the form.
 
 Basic usage example:
```
Comment (LinusU) [inline README.md]: Could you add an empty line here? Currently, line 25 and 26 is being rendered on the same line in the Markdown preview.

**Comment by LinusU** [comment]
Looks great, one last thing

**Comment by LinusU** [comment]
Thanks!

**Pull request merged**